src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -95,18 +95,18 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
     }
   };
 
+  const isOutgoingQuote = (quoteEntity: QuoteEntity): quoteEntity is OutgoingQuote => {
+    return quoteEntity.hash !== undefined;
+  };
+
   const onRetry = async (message: ContentMessage) => {
     const firstAsset = message.getFirstAsset();
 
     if (firstAsset instanceof Text) {
       const messageId = message.id;
       const messageText = firstAsset.text;
-      const mentions: MentionEntity[] = firstAsset.mentions() && firstAsset.mentions();
+      const mentions: MentionEntity[] = firstAsset?.mentions?.() || [];
```

Comment:
is this optional chaining really needed? firstAsset seem to be defined in this context and `.mentions()` seem to always return an array 🤔

Reply:
You should be correct, pretty sure the chaining made sense at some point, but there's been some change along the way, let me investigate